inductiva/structures/post_processing/fenicsx.py

```diff
@@ -0,0 +1,149 @@
+"""Post-process FEniCSx simulation outputs."""
+
+import os
+import tempfile
+
+import pyvista as pv
+
+import inductiva
```

Comment:
it is weird that you are inside the package inductiva but you are importing it, this seems like a circular import... No?

Reply:
I think it is the usual practice - Google Style Guide quote at 2.2 "Do not use relative names in imports. Even if the module is in the same package, use the full package name. This helps prevent unintentionally importing a package twice."

Reply:
doing `from inductiva import z` where z is a submodule would also adhere to the style guide, but we've been getting feedback to do it like this, so this is how it looks like almost everywhere now: https://github.com/inductiva/inductiva/pull/439#discussion_r1286626468

Reply:
@hpenedones, I'm following Luis's comment in the last PR (check out @luispcunha's comment with the link). Do you think it's better to approach it differently?